target/linux/mediatek/dts/mt7981b-yuncore-ax835.dts

```diff
@@ -0,0 +1,270 @@
+// SPDX-License-Identifier: (GPL-2.0 OR MIT)
+
+/dts-v1/;
+
+#include "mt7981.dtsi"
+
+/ {
+	compatible = "yuncore,ax835", "mediatek,mt7981";
+	model = "YunCore AX835";
+
+	aliases {
+		ethernet0 = &gmac0;
+		label-mac-device = &gmac0;
+		led-boot = &led_system;
+		led-failsafe = &led_system;
+		led-running = &led_system;
+		led-upgrade = &led_system;
+		serial0 = &uart0;
+	};
+
+	chosen {
+		// TODO from stock:
+		// bootargs = ".. loglevel=8 earlycon=uart8250,mmio32,0x11002000";
+		// u-boot,bootconf = "config-1";
+		stdout-path = "serial0:115200n8";
+	};
+
+	gpio-keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&pio 1 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		// TODO stock has:
+		// onekey: gpio5 active_high (JTAG JTDI - according to spreadsheet)
+		// "Higher level circuit to enable all LED light; Lower level circuit to close"
+
+		led_system: led@0 {
+			label = "red:system";
+			gpios = <&pio 4 GPIO_ACTIVE_LOW>;
+		};
+
+		led@1 {
+			label = "green:wifi2";
+			gpios = <&pio 34 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy0tpt";
+		};
+
+		led@2 {
+			label = "blue:wifi5";
+			gpios = <&pio 35 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy1tpt";
+		};
```

Comment:
Add missing led _hwwatchdog_
```suggestion
		};
		hwwatchdog: led@3 {
			label = "hwwatchdog";
			gpios = <&pio 7 GPIO_ACTIVE_LOW>;
			linux,default-trigger = "timer";
			led-pattern = <1000>, <1000>;
		};
```

Reply:
Good idea using the LED timer!
I've added you as a co-author ;-)